tools/cli/pkg/command/orchestration.go

```diff
@@ -134,6 +134,40 @@ Operations:
 {{- end }}
 `
 
+var kymaUpgradePreviewTpl = `Kyma Upgrade Preview
+Strategy:         {{.Parameters.Strategy.Type}}
+Schedule:         {{.Parameters.Strategy.Schedule}}
+Workers:          {{.Parameters.Strategy.Parallel.Workers}}
+Kyma Version:     {{with .Parameters.Kyma}}{{.Version}}{{end}}
```

Comment:
If .Parameters.Kyma.Version is not empty, it should be displayed. Otherwise this should be displayed:

`Kyma Version: <determined after start>`